.github/actions/aws-ci/action.yml

```diff
@@ -0,0 +1,74 @@
+name: 'AWS CI'
+description: 'AWS integration tests'
+inputs:
+  elk-version:
+    description: 'ELK version'
+    required: true
+  aws-access-key-id:
+    description: 'AWS access key id'
+    required: true
+  aws-secret-access-key:
+    description: 'AWS secret access key'
+    required: true
+  aws-account-type:
+    description: 'AWS account type'
+    required: false
+    default: single-account
+runs:
+  using: composite
+  steps:
+    - name: Build cloudbeat binary
+      shell: bash
+      run: mage -v build
+
+    - name: Run elasticsearch
+      uses: elastic/elastic-github-actions/elasticsearch@master
+      with:
+        stack-version: ${{ inputs.elk-version }}
+        security-enabled: false
+
+    - name: Elasticsearch disable dynamic mapping for findings
+      shell: bash
+      run: ./.ci/scripts/disable_dynamic_mapping.sh
+
+    - name: Run cloudbeat in background
+      env:
+        ES_HOST: http://localhost:9200
+        ES_USERNAME: elastic
+        ES_PASSWORD: changeme
+        AWS_ACCESS_KEY_ID: ${{ inputs.aws-access-key-id }}
+        AWS_SECRET_ACCESS_KEY: ${{ inputs.aws-secret-access-key }}
+        AWS_ACCOUNT_TYPE: ${{ inputs.aws-account-type }}
+      shell: bash
+      run: |
+        ./cloudbeat -c deploy/aws/cloudbeat-aws.yml -d '*' &
+
+    - name: Wait for cloudbeat to send some events
+      shell: bash
+      run: sleep 20
```

Comment:
20 second seems not a lot (?). Hard to say, right. But I think 20s can easily add to flakyness. 

I also see other actions (e.g. azure-ci) wait a different amount of seconds. Maybe good to have a const or something where we can easily bump all the wait periods? Or at least have all the pipelines with the same await period

Reply:
https://github.com/elastic/cloudbeat/blob/6db9c916e2d91b16cf7fc65554284c395e2adceb/.github/workflows/aws-ci.yml#L66

It's `264` in the current workflow, and I tried to reduce it while ensuring that the pytest finds rules results (same goes to other runs as well)